specs/Qowaiv.CodeAnalysis.Specs/Cases/DefinePropertiesAsImmutables.cs

```diff
@@ -127,7 +127,7 @@ namespace Noncompliant
 {
     public class Class
     {
-        public int PublicSet { get; set; } // Noncompliant {{Remove this setter, or make this property init-only.}}
+        public int PublicSet { get; set; } // Noncompliant {{Remove this setter, make this property init-only, or decorate the type being mutable.}}
```

Comment:
I would suggest either "mark as (being) mutable" or "decorate with mutable attribute".
F.e. (take your pick):
"or decorate the type with the mutable attribute."
"or mark the type as being mutable."

Reply:
Like this?